model_compression_toolkit/gptq/README.md

```diff
@@ -0,0 +1,28 @@
+# Gradient-based Post-Training Quantization (GPTQ)
+
+The GPTQ package allows to run advanced quantization algorithm, in which the quantized parameters rounding is optimized after the initial quantization is applied.
```

Comment:
The English here is not correct. This is an example of one way to revise it:

GPTQ is a core feature of MCT, offering an advanced quantization algorithm that optimizes parameter rounding after the initial quantization process. It preserves the principles of post-training quantization, utilizing a small, unlabeled dataset without the need for additional model information.

Reply:
That's why I asked you to review this PR, thanks (;